dev/com.ibm.ws.ras.instrument_test/test/com/ibm/ws/ras/RasTransformTest.java

```diff
@@ -34,25 +39,89 @@
 
 import com.ibm.ws.ras.instrument.internal.main.LibertyRuntimeTransformer;
 
+@SuppressWarnings("deprecation")
```

Comment:
This test doesn't actually check to make sure that classes are instrumented with trace.  Should there be something added to make sure for a method there is an entry and exit and that the annotations are there?

Reply:
Correct, I'll leave that for someone who actually knows how this class works :)